datalab_cohorts/expectation_generators.py

```diff
@@ -40,26 +41,29 @@ def generate_dates(population, earliest_date, latest_date, rate):
     (increasingly common)
 
     """
-    assert (
-        rate == "exponential_increase"
-    ), "Only exponential increase currently supported"
     low = datetime.strptime(earliest_date, "%Y-%m-%d").date()
     high = datetime.strptime(latest_date, "%Y-%m-%d").date()
     elapsed_days = (high - low).days
 
-    # We oversample the distribution to trim the long tail of the
-    # exponential function
-    oversample_ratio = 1.5
-    distribution = (
-        expon.rvs(loc=0, scale=0.1, size=int(population * oversample_ratio))
-        * elapsed_days
-    ).astype("int")
-    distribution = distribution[distribution <= elapsed_days]
+    if rate == "exponential_increase":
+        # We oversample the distribution to trim the long tail of the
+        # exponential function
+        oversample_ratio = 1.5
+        distribution = (
+            expon.rvs(loc=0, scale=0.1, size=int(population * oversample_ratio))
+            * elapsed_days
+        ).astype("int")
+        distribution = distribution[distribution <= elapsed_days]
+    elif rate == "uniform":
+        distribution = uniform.rvs(size=int(population)) * elapsed_days
+        distribution = distribution.astype("int")
+    else:
+        print("Only exponential and uniform distributions currently supported")
```

Comment:
I'd explicitly say `exponential_increase` here both (a) because we could imagine decreases in the future and (b) because that's the actual keyword (I realise that the underscore wasn't there previously!)

Reply:
have done